src/plugins/DocsTargetBranch/docs_target_branch.ts

```diff
@@ -0,0 +1,96 @@
+import { PRContext } from "../../types";
+import { Application } from "probot";
+import { filterEventByRepo } from "../../util/filter_event_repo";
+import { filterEventNoBot } from "../../util/filter_event_no_bot";
+import { REPO_HOME_ASSISTANT_IO, ORG_HASS } from "../../const";
+import {
+  extractIssuesOrPullRequestMarkdownLinks,
+  extractPullRequestURLLinks,
+} from "../../util/text_parser";
+
+const NAME = "DocsTargetBranch";
+const SKIP_REPOS = ["brands"];
+
+export const bodyShouldTargetCurrent: string =
+  "It seems that this PR is targeted against an incorrect branch. Documentation updates which apply to our current stable release should target the `current` branch. Please change the target branch of this PR to `current` and rebase if needed.";
+export const bodyShouldTargetNext: string =
+  "It seems that this PR is targeted against an incorrect branch since it has a parent PR on one of our codebases. Documentation that needs to be updated for an upcoming release should target the `next` branch. Please change the target branch of this PR to `next` and rebase if needed.";
+
+export const initDocsTargetBranch = (app: Application) => {
+  app.on(
+    "pull_request.opened",
```

Comment:
You only filter on when pull requests are opened, not on when they are updated ?

Reply:
The way I read the issue I only thought about opened events, added `pull_request.edited` as well now 👍